internal/packer_registry/env/env.go

```diff
@@ -36,6 +36,18 @@ func HasHCPCredentials() bool {
 }
 
 func IsPAREnabled() bool {
+
+	// HCPCredentials being set is a requirement for HCP Packer Registry communication.
+	// If not set we can quickly say Packer is not PAREnabled.
+	if !HasHCPCredentials() {
+		return false
+	}
+
+	// If we have HCPCredentials and HCPPackerRegistry is not set should PAR be enabled?
```

Comment:
@sylviamoss what are you thoughts here.

Reply:
I think I prefer this one to not check for HCP Credentials and let it fail when trying to create the client. The reason for that is that one might enable HCP Packer Registry but it won't fail for missing credentials because if `IsPAREnabled == false` it will never try to create the client. 

In other words, HCP Packer Registry might be enabled with missing or wrong configuration and it's interesting to fail in that case.

Reply:
Got it. I added because I had a failing test which I thought was because of the HCPCredentials but we can repro so I'm going to drop the commit. thanks!